eo.txt

```diff
@@ -1394,9 +1396,13 @@ server_packet(Avatar, Remove)
 "Player has appeared in nearby view"
 server_packet(Players, Agree)
```

Comment:
Does this really work? I'm pretty sure I captured the data from GameServer and the `break` and `1` are present.

Reply:
The client expects a `NearbyInfo` struct and reads the garbage data send by GameServer due to a fluke of the weird packet-chunking behavior in the official packet reader.
Relevant conversation on this subject:
![image](https://user-images.githubusercontent.com/22178573/202830933-e39fc7e4-17a1-4822-a88d-4bb2d31fc45a.png)
![image](https://user-images.githubusercontent.com/22178573/202830943-47fb8cd5-abab-45d6-98b1-9008d9a2eaf4.png)
![image](https://user-images.githubusercontent.com/22178573/202830958-3273ffac-6f30-49f4-873a-1a4abf7e7f8b.png)
![image](https://user-images.githubusercontent.com/22178573/202830975-3ba77bf1-5ee4-493f-9afb-70ef7b3853b7.png)
![image](https://user-images.githubusercontent.com/22178573/202831005-b931824d-85bd-463e-b679-5e2dc0f6ff93.png)
![image](https://user-images.githubusercontent.com/22178573/202831074-9f27f973-d8c3-4d16-8796-ce08146ba9be.png)

Reply:
More relevant conversation:
![image](https://user-images.githubusercontent.com/22178573/202851257-a4766716-6415-4697-bf0d-2b36267151c8.png)
![image](https://user-images.githubusercontent.com/22178573/202851229-842cb73e-dbfe-4548-8422-49687cfbdeca.png)